generator.rb

```diff
@@ -0,0 +1,104 @@
+# from hw document, to find a vowel
+def is_vowel?(letter)
+  case letter
+    when "a", "e", "i", "o", "u", "A", "E", "I", "O", "U" 
+      return true
+    else 
+        return false
+    end
+end
+
+# determines if there is a vowel in a word
+def has_vowel(word)
+count = 0
+  word.each_char do |letter|
+    if is_vowel?(letter)
+        count += 1
+    else
+        count = count
+    end
+  end
+  
+  if count > 0
+    return true
+  else 
+    return false
+  end
+end
+
+# my attempt at getting the game to play again
+play_game = true
+
+while play_game
+
+    def run_generator
+      # To make sure that the input is greater than one character and contains a vowel
+        puts "Please enter the first word:"
+        first_word = gets.chomp
+        until first_word.length > 1 && has_vowel(first_word)
+          puts "Your word must be > 1 character & must have a vowel: "
+          first_word = gets.chomp
+        end
+        
+        puts "Please enter the second word:"
+        last_word = gets.chomp
+        until last_word.length > 1 && has_vowel(last_word)
+          puts "Your word must be > 1 character & must have a vowel: "
+          last_word = gets.chomp
+        end
```

Comment:
the indentation between line 36 and 48 is inconsistent with the rest of the file. Best practice is to rely on your tab button; even if it isn't 2 spaces right now, it's better if they are consistent with each other. 

Command comma should open your settings, btw, you don't need a special plugin to adjust tab spacing. Just search for 'tab' when you have the settings editor open and you should find it.